src/plays/react-todo-app/react-todo.css

```diff
@@ -0,0 +1,145 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.play-details-body {
+    background-image: linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%);
+    color: white;
```

Comment:
You must not override the style for `.play-details-body` as it will impact other plays.

Your UI element should be wrapped in a div and have a class to get the style done. Here is an example it impacted another play

![image](https://user-images.githubusercontent.com/3633137/165014925-6cc35544-a980-46fa-b8cc-08d06dc8d7f2.png)

Reply:
Should I create another pull request?

Reply:
No Why? Please make your changes and push on the same branch and pr